Simulator/Recorders/Recordable.h

```diff
@@ -45,30 +55,21 @@ template <typename T> class Recordable : public RecordableBase {
 
    /// Get the value of the recordable variable at the specified index.
    /// return A variant representing the recorded value (uint64_t, double, or string)
-   virtual variant<uint64_t, double, string> getElement(int index) const override
+   virtual variantTypes getElement(int index) const override
    {
       return eventTimeSteps_[index];
    }
 
-   /// Get A string representing the data type of the recordable variable
-   virtual const string &getDataType() const override
-   {
-      return basicDataType_;
-   }
-
    ///  Cereal serialization method
    template <class Archive> void serialize(Archive &archive)
    {
       archive(cereal::virtual_base_class<RecordableBase>(this),
-              cereal::make_nvp("eventTimeSteps_", eventTimeSteps_),
-              cereal::make_nvp("basicDataType_", basicDataType_));
+              cereal::make_nvp("eventTimeSteps_", eventTimeSteps_));
```

Comment:
I think this variable name needs to be changed to something more generic than "event...".

Reply:
renamed it from eventTimeSteps_ to dataSeries_ in the RecordableVector class.